index.bs

```diff
@@ -260,7 +260,7 @@ spec:url; type:dfn; text:scheme
 
     1. Let |portalBrowsingContext| be the [=guest browsing context=] of [=this=].
 
-        If no such context exists, throw an "{{InvalidStateError}}" {{DOMException}}.
+        If it is null, throw an "{{InvalidStateError}}" {{DOMException}}.
```

Comment:
The various null check steps now seem better as separate steps, instead of as two paragraphs in one step. But, that's a stylistic choice, up to you.

Reply:
Done.